src/titiler/core/titiler/core/algorithm/dem.py

```diff
@@ -37,12 +37,10 @@ def __call__(self, img: ImageData) -> ImageData:
         shaded = numpy.sin(altituderad) * numpy.sin(slope) + numpy.cos(
             altituderad
         ) * numpy.cos(slope) * numpy.cos(azimuthrad - aspect)
-        hillshade_array = 255 * (shaded + 1) / 2
-
-        data = numpy.expand_dims(hillshade_array, axis=0).astype(dtype=numpy.uint8)
+        data = 255 * (shaded + 1) / 2
 
         if self.buffer:
-            data = data[:, self.buffer : -self.buffer, self.buffer : -self.buffer]
+            data = data[self.buffer : -self.buffer, self.buffer : -self.buffer]
```

Comment:
we keep the data as 1d, and it will then be expanded when creating the ImageData object